src/card.css

```diff
@@ -0,0 +1,113 @@
+.container {
+  box-sizing: border-box;
+  max-width: 200px;
+
+  background: #fff;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding-top: 32px;
+  padding-bottom: 16px;
+}
+
+.container__image {
+  display: flex;
+}
+
+.image {
+  margin: 0 19px;
+  width: 160px;
+  height: 134px;
+  object-fit: cover;
+  object-position: center;
+  margin-bottom: 40px;
+}
+
+.container__info {
+  padding: 0 16px;
+  box-sizing: border-box;
+}
+
+.container__heading {
+  margin-bottom: 4px;
+
+  display: flex;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.container__text-small {
+  color: #616070;
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.stars__reviews {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  font-size: 10px;
+  font-weight: 400;
+  color: #060b35;
+  line-height: 14px;
+}
+
+.container__price {
+  display: flex;
+  justify-content: space-between;
+  line-height: 18px;
+
+  font-size: 700;
+
+  margin-bottom: 16px;
+}
+
+.container__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  
+  display: flex;
+  align-items: flex-end;
+}
+
+.cointainer__price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.container__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 164px;
+  height: 38px;
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  transition: color 0.5s;
```

Comment:
would be great to add all style form hover to transition (border-color, color, background-color)